### ISSUE TITLE
Move `Roles` and `Permissions` to new `Admin` section

### ISSUE THREAD

**Issue by Willjobs94** [RVelopers/TanitaTraker-NetCore#38]
We should move out the **Roles** and **Permissions** options to a new section called **Administration**

![image](https://user-images.githubusercontent.com/10265964/41134526-cabf8efc-6a9a-11e8-96e3-53102306db51.png)

**Comment by christianrd** [comment]
Good idea @Willjobs94, we can rename the users list by Administration or configuration...

**Comment by Willjobs94** [comment]
Not, we should create another section for the configuration or administration

**Comment by christianrd** [comment]
Umm ok, that's better...
